src/board/v3/nap/nap_hw.h

```diff
@@ -68,6 +98,11 @@ typedef struct {
 #define NAP_ACQ_FFT_CONFIG_SCALE_Pos (1U)
 #define NAP_ACQ_FFT_CONFIG_SCALE_Msk (0x3FFFFFFFU << NAP_ACQ_FFT_CONFIG_SCALE_Pos)
 
+#define NAP_TRK_CONTROL_PRN_Pos (3u)
```

Comment:
Should probably be SAT. I'm also going to change the naming in the register map to match this.